search/django/weedid/views.py

```diff
@@ -31,7 +32,9 @@ def upload(request):
         user_id = request.user.id
         images = []
         file_weedcoco = request.FILES["weedcoco"]
-        for image_reference in json.load(file_weedcoco)["images"]:
+        weedcoco_json = json.load(file_weedcoco)
+        validate(weedcoco_json)
```

Comment:
I assume the errors will somehow be reported in the upload status, via the celery task...